spec.md

```diff
@@ -16,17 +16,30 @@ while not compromising on the semantics.
 # Overview
 
 This specification defines a number of events that can occur in the
-broker-bank relationship, and how these events should be transferred
-between two systems.
-
-The standard defines the behaviour of two actors, the producer and the
-consumer, as defined in the CloudEvents specification. The producer
-creates events and the consumer reads events created by the
-producer. In order to ensure an engaging end user experience both parties
-will act in both roles as defined by this standard.
-
-Additionally behaviour in this specification is defined on three
-levels of operations.
+broker - service provider relationship, and how these events should be
+transferred between two systems.
+
+The service provider is defined as the party providing a service,
+defined in this specification. This might for example be a loan, an
+insurance product or a mortgage.
```

Comment:
Hmm this standard only refers to loans, shouldn't we have a separate document for mortgages and insurances?

Reply:
While some data might be shared most is not, i am worried that we might create leaky or ambiguous abstractions.

Reply:
Well thats the same issue with different countries. What is shared is basically the substrate for transferring events

Reply:
Maybe we should create a folder called sweden and rename this file to sweden/loan.md?